src/auth.ts

```diff
@@ -0,0 +1,15 @@
+import dotenv from 'dotenv';
+import jwt from 'express-jwt';
+import fs from 'fs';
+dotenv.config();
+
+// Top-Level await would make this so much nicer
+if(process.env.KEY_FILE === undefined) {
+  process.env.KEY_FILE = 'key.rs';
```

Comment:
Suggestion: Print out an error which tells the user they need to add the key.rs file.